src/lib/auth/supabase-client.ts

```
@@ -0,0 +1,434 @@
+        return builder;
+      },
+      
+      // Execution operations
+      single: () => {
+        logOperation('single', query);
+        const result = filteredData.length > 0 ? filteredData[0] : null;
+        return Promise.resolve({ data: result, error: null });
+      },
+      
+      // Direct execution without additional modifiers
+      then: (onfulfilled: (value: { data: any, error: null }) => any) => {
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)